sagemaker-debugger/model_specific_realtime_analysis/bert_attention_head_view/bert_attention_head_view.ipynb

```diff
@@ -20,6 +20,16 @@
     "[1] *Visualizing Attention in Transformer-Based Language Representation Models*:  Jesse Vig, 2019, 1904.02679, arXiv"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        while description["SecondaryStatus"] != "Completed":
</pre><p>The above markdown says:</p><blockquote>We can access the tensors from S3 once the training job is in status Training or Completed.</blockquote><p>Your change will fix the out-of-index error, but will take a bit longer because the job must complete first, instead of reaching Training. You could consider/test fixing the list syntax by changing braces to brackets, such as:</p><pre class="ql-syntax" spellcheck="false">while description["SecondaryStatus"] not in ["Training", "Completed"]:
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3097/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Good point, I tried that. The problem is that at the 'Training' stage, there still may not be any data (i.e. training at the first epoch) so the CI error still occurs. After the first epoch is done, like the text says, there are no problems at all. I don't know if there is any way to track when the first epoch is done? What if we keep querying and try...except.. the error until it is gone?

Another concern I have about model not being fully trained is subsequent steps where all the tensors from every epoch is collected. I feel like the model still needs to be fully trained for those steps. Let me know if I am wrong.